Xamarin.Forms.Xaml.UnitTests/XamlLoaderCreateTests.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using NUnit.Framework;
+
+namespace Xamarin.Forms.Xaml.UnitTests
+{
+	[TestFixture]
+	public class XamlLoaderCreateTests
+	{
+		[Test]
+		public void CreateFromXaml ()
+		{
+			var xaml = @"
+				<ContentView xmlns=""http://xamarin.com/schemas/2014/forms""
+							 xmlns:x=""http://schemas.microsoft.com/winfx/2009/xaml""
+						 	 x:Class=""Xamarin.Forms.Xaml.UnitTests.FOO"">
+					<Label Text=""Foo""  x:Name=""label""/>
+				</ContentView>";
+
+			var view = XamlLoader.Create (xaml);
+			Assert.That (view, Is.TypeOf<ContentView> ());
```

Comment:
/cc @kallisto Can you follow up on this and make sure it's implemented as a matter of urgency. It's blocking your demo now.